cmd/agent_pay.go

```diff
@@ -110,12 +86,15 @@ func pay(cmd *cobra.Command, args []string, paymentType PaymentType, daemon bool
 		Amount:  payAmt.String(),
 		PayType: paymentType.String(),
 	}
-	if !daemon {
-		defer journal.Close()
-	}
+	defer journal.Close()
```

Comment:
This had to do with `autopilot` and (i think but not 100% sure) unexpected shutdowns of the daemon - how confident are you that we can remove this `if daemon` conditional without consequences in autopilot?

Reply:
I don't actually remember removing that code. I might be missing some commits...